modules/mcagar/src/main/ts/ephox/mcagar/api/bdd/TinySelections.ts

```diff
@@ -31,5 +40,6 @@ export {
   select,
   setCursor,
   setSelection,
+  setRawSelection,
```

Comment:
McAgar changelog entry?

Reply:
I didn't even notice this was in mcagar. oops.